src/wp-admin/js/inline-edit-post.js

```diff
@@ -134,16 +191,16 @@ inlineEditPost = {
 			fields.push('post_parent', 'page_template');
 		}
 
-		// add the new edit row with an extra blank row underneath to maintain zebra striping.
+		// Add the new edit row with an extra blank row underneath to maintain zebra striping.
 		editRow = $('#inline-edit').clone(true);
 		$( 'td', editRow ).attr( 'colspan', $( 'th:visible, td:visible', '.widefat:first thead' ).length );
 
 		$(t.what+id).removeClass('is-expanded').hide().after(editRow).after('<tr class="hidden"></tr>');
 
-		// populate the data
+		// Populate the data
 		rowData = $('#inline_'+id);
 		if ( !$(':input[name="post_author"] option[value="' + $('.post_author', rowData).text() + '"]', editRow).val() ) {
-			// author no longer has edit caps, so we need to add them to the list of authors
+			// The post author no longer has edit caps, so we need to add them to the list of authors
```

Comment:
Please don't abbreviate words. Also missing a period at the end of the sentence.